api/envoy/config/listener/v3/listener_components.proto

```diff
@@ -269,11 +254,6 @@ message FilterChain {
   // <envoy_v3_api_field_config.listener.v3.Listener.filter_chain_matcher>`
   // requires that filter chains are uniquely named within a listener.
   string name = 7;
-
-  // [#not-implemented-hide:] The configuration to specify whether the filter chain will be built on-demand.
-  // If this field is not empty, the filter chain will be built on-demand.
-  // Otherwise, the filter chain will be built normally and block listener warming.
-  OnDemandConfiguration on_demand_configuration = 8;
```

Comment:
We now basically won't remove any field. I think.

I think you only need to fix annotation of `[#not-implemented-hide:]`. This annotation should be in single line. And you can also add deprecated tag to the field.

A field with `[#not-implemented-hide:]` annotation and `deprecated` tag will be ignored forever in both doc and code.

Reply:
If it was never implemented I think it's OK to remove/reserve it.

Reply:
then, okay.